draft-ietf-mptcp-rfc6824bis-11.xml

```diff
@@ -2050,7 +2048,7 @@ subflow-specific windows.</t>
         <c>This document, <xref target="sec_init"/></c>
 
         <c>C</c>
-        <c>Do not attempt to connect to source address</c>
+        <c>Don't reuse source address for new subflows.</c>
```

Comment:
That wording is not very clear to me. How about:

"Do not attempt to establish new subflows to the source address"

Reply:
Ok, used your suggestion.